specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -9675,28 +10562,23 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/PremierAddOn"
+              "$ref": "#/definitions/FunctionSecrets"
```

Comment:
Isnt this a breaking change where the API signature and the response is changing? GET becomes a POST /listsyncfunctiontriggerstatus